### PR TITLE
Fix method to return timestamp as minutes since epoch in dev & test

### PR DESCRIPTION
**Pull request by pavel-yakubovski-tr**
It seems like this method https://github.com/queue-bus/queue-bus/blob/master/lib/queue_bus/heartbeat.rb#L50 returns "seconds since epoch" instead of "minutes since epoch" in dev & test environments. As a result, heartbeat events are not really fired on local.

I'm intentionally not adding any tests, as there is already a failing test that is fixed by this change.

### CODE REVIEW

**Review by aalong-tr — APPROVED** [review]

**Review by joeellis — APPROVED** [review]
@pavel-yakubovski-tr Great find and makes sense - were you able to confirm that the locking issue didn't return with this change? Also, can we update the `CHANGELOG.md` and version here as well? `0.9.1` seems reasonable here to me, and @kbacha can likely handle the rubygem push after it's merged.

**Comment by kbacha** [comment]
@pavel-yakubovski-tr @joeellis Just add an entry to the Unreleased section: https://github.com/queue-bus/queue-bus/blob/master/CHANGELOG.md#unreleased

I'll merge and release a new version after that.

**Comment by kbacha** [comment]
@pavel-yakubovski-tr I've updated trunk to run circle, but I also ran rubocop. Would you mind bringing this branch up to date?

**Comment by pavel-yakubovski-tr** [comment]
> @pavel-yakubovski-tr I've updated trunk to run circle, but I also ran rubocop. Would you mind bringing this branch up to date?

@kbacha updated!

**Comment by pavel-yakubovski-tr** [comment]
> @pavel-yakubovski-tr Great find and makes sense - were you able to confirm that the locking issue didn't return with this change? Also, can we update the `CHANGELOG.md` and version here as well? `0.9.1` seems reasonable here to me, and @kbacha can likely handle the rubygem push after it's merged.

@joeellis it's a bit tricky to confirm because we don't have exact steps to reproduce, but at first glance it looks like it is fixed.

**Comment by joeellis** [comment]
@pavel-yakubovski-tr 👍 That's enough for me for now then - mostly a check to make sure it isn't somehow making things worse / locking in new unexpected ways. Also less concerned here since this will only affect dev / test envs.

**Pull request merged**
